structwrapping/structwrapping_test.go

```diff
@@ -0,0 +1,165 @@
+package structwrapping
+
+import (
+	"testing"
+
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/aead"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStructWrapping(t *testing.T) {
+	wrapper := aead.NewWrapper(nil)
+	wrapper.SetConfig(map[string]string{"aead_type": "aes-gcm", "key": "QmjueU/LMsZAO+rvSUMcpkBziCD5ON7BgxVqcZ6+TCI="})
```

Comment:
perhaps check for err here... it might save a future dev some time tracking down why the tests are failing.

Reply:
Done